lib/index.js

```diff
@@ -104,6 +104,8 @@ function render(loaded, props) {
 function createLoadableComponent(loadFn, options) {
   var _class, _temp;
 
+  var throwOnError = "throwOnError" in options ? options.throwOnError : true;
```

Comment:
@somewhatabstract is this how you were thinking the `thowOnError` option would be specified?

Reply:
Yeah, seems reasonable to me :)